srml/balances/src/lib.rs

```diff
@@ -238,7 +387,7 @@ decl_module! {
 	}
 }
 
-// For funding methods, see Currency trait
+/// The functions in this implementation affect storage.
```

Comment:
well not vesting_balance,
and idea: something doable is to make to implement block on with only immutable functions and this doc another with mutables and another doc.
```rust
/// immutables
impl<T: Trait<I>, I: Instance> Module<T, I> {
...
}
/// mutables
impl<T: Trait<I>, I: Instance> Module<T, I> {
...
}
```

Reply:
Removing this comment, the functions inside the `impl` are divided as
```
// PUBLIC IMMUTABLES
...
// PRIVATE MUTABLES
...
```

Reply:
We can divide `impl`s that way. Going to keep this PR limited to comments and not affect code. That does leave us with potentially four `impl`s in a module: {pub, priv}{mut, immut}

Reply:
ok
as the division is only for documentation purpose, private could go in whatever block without any consistency imo.